client-vms/src/util.ts

```diff
@@ -1,3 +1,4 @@
+import { createHash } from "node:crypto";
```

Comment:
Won't this cause the client to stop working on the browser because this is a `nodejs` library?

Reply:
I actually need to remove this util.